config/config.go

```diff
@@ -349,6 +349,16 @@ type Local struct {
 	// and is expected to happen only on either startup ( after enabling the catchpoint interval, or on certain database upgrades ) or during fast catchup. The values specified here
 	// and their meanings are identical to the ones in LedgerSynchronousMode.
 	AccountsRebuildSynchronousMode int `version[12]:"1"`
+
+	// MaxCatchpointDownloadTimeSec defines the maximum amount of time a client will be keeping the outgoing connection of a catchpoint download request open for processing before
+	// shutting it down. Networks that has large catchpoint files, slow connection or slow storage could be a good reason to increase this value. Note that this is a client-side only
+	// configuration value, and it's independent of the actual catchpoint file size.
+	MaxCatchpointDownloadTimeSec uint64 `version[13]:"7200"`
+
+	// MinCatchpointFileDownloadRate defines the minimal download speed that would considered to be "acceptable" by the catchpoint file fetcher, measured in bytes per seconds. If the
+	// provided stream speed drops below this threshold, the connection would be recycled. Note that this field is evaluated per catchpoint "chunk" and not on it's own. This field should
+	// always be non-zero, or the default of 20480 would be used.
+	MinCatchpointFileDownloadRate uint64 `version[13]:"20480"`
```

Comment:
To be consistent with the server, and eliminate the ambiguity of the units, I prefer to call this MinCatchpointFileDownloadBytesPerSecond.